openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -183,7 +190,7 @@ def verify_error_response(field_name, data):
                 "Field '{0}' cannot be edited.".format(field_name), data["field_errors"][field_name]["user_message"]
             )
 
-        for field_name in ["username", "email", "date_joined", "name"]:
+        for field_name in ["username", "date_joined"]:
```

Comment:
"email" is still listed as a read_only_field in AccountUserSerializer. Should that be updated?